opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/Adis.java

```diff
@@ -1098,7 +1103,9 @@ public ProlongResult prolong(String media, Account account, int useraction,
             }
         }
         form.add(new BasicNameValuePair(media.split("\\|")[0], "on"));
-        form.add(new BasicNameValuePair("textButton$1",
+        // Stuttgart: textButton$0, others: textButton$1
+        form.add(new BasicNameValuePair(
+                doc.select("input[value=Markierte Titel verlängern]").attr("name"),
```

Comment:
Can we have a fallback to textButton$1 in case this select() does not return anything? I imagine other libraries might have other labels or just broken encoding around the ä ;) The same goes for the other places with similar selects.

Reply:
Makes sense, done in 01999f51e.